AirCasting/CreateSessionViews/TurnOnLocation/TurnOnLocationViewModel.swift

```diff
@@ -45,22 +49,35 @@ class TurnOnLocationViewModel: ObservableObject {
             showRequestLocationAlert()
         } else {
             switch process {
-            case .sdClear:
-                restartABLink.toggle()
-            case .sdSync:
-                unplugABLink.toggle()
+            case .sdClear, .sdSync:
+                handleBluetoothOrToggle(for: process)
+                
             case .createSession(let sessionContext):
                 if sessionContext.isMobileSession {
-                    isMobileLinkActive = true
-                } else if bluetoothHandler.isBluetoothDenied() {
-                    isTurnBluetoothOnLinkActive = true
+                    isProceedToSelectDeviceTypeLinkActive = true
```

Comment:
This is here essentially because the UI is structured in the way that when you go to create a mobile session:
1. You are asked for location permission
2. You choose the device type (Mic or AB)
3. You are asked for bluetooth permission if you've chosen AB